submissions/aymkin/HTML-CSS-Popup/style.css

```diff
@@ -0,0 +1,276 @@
+/* NAV START */
+.container {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    width: 340px;
+    position: absolute;
```

Comment:
Not a good idea to define layout by `position: absolute`.

Reply:
why that is not a good idea?
what is your suggestion?